### PR TITLE
Laravel 10 support

### PR DESCRIPTION
**Pull request by truestealth**

### CODE REVIEW

**Comment by stevebauman** [comment]
Thanks @truestealth! Will merge and make new release once tests are green 👍

File: composer.json [hunk 1]
```diff
@@ -33,10 +33,10 @@
         "ext-ldap": "*",
         "ext-json": "*",
         "psr/log": "^1.0|^2.0|^3.0",
-        "psr/simple-cache": "^1.0|^2.0",
+        "psr/simple-cache": "^1.0|^2.0|^3.0",
```
Comment (stevebauman) [inline composer.json:36]:
```suggestion
        "psr/simple-cache": "^1.0|^2.0",
```

Comment (stevebauman) [reply composer.json:36]: Needs to be kept to `^1.0|^2.0` due to: https://github.com/DirectoryTree/LdapRecord-Laravel/issues/392

**Comment by truestealth** [comment]
@stevebauman Thanks

**Pull request merged**